lib/xapi/implementation.rb

```diff
@@ -91,6 +91,8 @@ def assemble_readme
 #{readme_body}
 
 #{problem.source_markdown}
+
+#{incomplete_solutions_body}
```

Comment:
Does this warrant it's own section? Is this the right place to put this section, at the end of the file?
Maybe it should go in the `readme_body` with the rest of the information about submitting a solution.